style.css

```diff
@@ -0,0 +1,273 @@
+  text-align: center;
+}
+
+.works_content h3 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding: 0 25px;
+  margin-bottom: 15px;
+}
+
+.works_cards .card {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  outline: 1px solid #eee;
+  border-radius: 7px;
+}
+
+.card_image {
+  height: 50%;
+}
+
+.card_image img {
+  height: 100%;
+  width: 100%;
+}
+
+.card_content {
+  height: 50%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.card_content h3 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+}
+
+.card_content ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  gap: 10px;
+}
+
+.card_content ul li {
+  background-color: #eee;
+  padding: 8px 12px;
+  font-size: 12px;
+  color: #3a4a42;
+}
+
+/* About */
+
+/* .about {
+  margin-top: 25px;
+}
+
+.about_content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.about_content h3 {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding-right: 25px;
+  margin-bottom: -40px;
+}
+
+.about_content p {
+  color: #42526e;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  margin-bottom: 15px;
+  padding: 0 15px;
+}
+
+.about_boxes {
+  margin-top: 50px;
+}
+
+.box {
+  width: 327px;
+  height: 316px;
+  background: #ebf0ee;
+  margin-bottom: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 10px;
+}
+
+.box h3 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #344563;
+}
+
+.box ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+  gap: 10px;
+}
+
+.box ul li {
+  background: #fff;
+  padding: 8px 12px;
+  font-size: 12px;
+  color: #36b37e;
+  border-radius: 8px;
+} */
+
+@media screen and (min-width: 375px) and (max-width: 768px) {
+  .headline {
+    background-image: url("logo/header-illsutration-mobile.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 653.32px;
+  }
+
+  .headline_content {
+    margin-top: 17%;
+  }
+
+  /* .works_content h1 {
+    padding: 0 30px;
+    margin-bottom: 25px;
+  }
+
+  .works_cards {
+    margin-top: 25px;
+    display: grid;
+    grid-template-columns: auto;
+    padding: 5%;
+    grid-template-rows: repeat(6, 475px);
+    gap: 20px;
+  }
+
+  .about_content {
+    background-image: url("Assets/about\ me-illustration-mobile.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 350.38px;
+  }
+
+  .box img {
+    width: 160px;
+    margin-bottom: -50px;
+  } */
+}
+
+/* .illustration1 {
+  display: flex;
+  align-items: center;
+  justify-content: end;
+} */
```

Comment:
[OPTIONAL]
- [ ] Kindly delete this block of commented code, as it affects badly the organization and look of your code.